RadarSDK.xcodeproj/xcshareddata/xcschemes/RadarSDKTests.xcscheme

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
```

Comment:
Add a new schema for `RadarSKDTests` target, so we could do the testing within the xcode, with the benefit of individual case testing and debugging.

Reply:
this is huge!!